docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  mongodb:
+    image: bitnami/mongodb:4.4.15
+    platform: linux/amd64
+    environment:
+      MONGODB_USERNAME: pritunl
+      MONGODB_DATABASE: pritunl
+      MONGODB_PASSWORD: pritunl
+      MONGODB_ROOT_PASSWORD: pritunl
+      MONGODB_EXTRA_FLAGS: --wiredTigerCacheSizeGB=0.5
+  pritunl:
+    build:
+      context: .
+    platform: linux/amd64
+    cap_add:
+      - NET_ADMIN
+    ports:
+      - 8443:443
```

Comment:
Is it self-signed SSL here? Is nginx+certbot still a PITA via docker-compose ? it may be out of the scope of this docker-compose :)

Reply:
Yeah, it is a good idea switching to plain HTTP for testing purposes. User will always be able to change that.

Reply:
pritunl automatically generates self-signed certificate. I tried different methods to disable HTTPS, but none of them worked. So i think it will be better to leave it as is. And yeah, nginx+certbot is out of scope here.